src/utils/common.ts

```diff
@@ -88,7 +87,7 @@ export function nativeToToken(
 ): string {
   const priceWei = toBNWei(price);
   const toAmount = toBNWei(fromAmount).div(priceWei);
-  return ConvertDecimals(nativeDecimals, toDecimals)(toAmount);
+  return ConvertDecimals(nativeDecimals, toDecimals)(toAmount.toString()).toString();
```

Comment:
Explanation: there are actually two variants of `ConvertDecimals()` - one supporting `Web3.js BN` and the other supporting `ethers BigNumber`. It's not strictly required because raw number and string types are used on both input and output sides, but there's a constraint in that `FormattingUtilsInternal.ts` is not intended to import from other files, and it ideally shouldn't depend directly on `ethers BigNumber` either.

So I propose that we stick with the Web3 variant hidden in `FormattingUtilsInternal.js` and clean this up post-import.

Reply:
Update - it was possible to drop the duplicate imports after merging in the latest changes to master.